BadMedicine.Dicom.Tests/DicomDataGeneratorTests.cs

```diff
@@ -86,6 +86,32 @@ public void Test_CreatingInMemory_ModalityCT()
 
             generator.Dispose();
             
+        }
+        [Test]
+        public void Test_Anonymise()
+        {
+            var r = new Random(23);
+            var person = new Person(r);
+
+            var generator = new DicomDataGenerator(r,new DirectoryInfo(TestContext.CurrentContext.WorkDirectory),"CT");
+            
+            // without anonymisation (default) we get the normal patient ID
+            var ds = generator.GenerateTestDataset(person, r);
+            
+            Assert.IsTrue(ds.Contains(DicomTag.PatientID));
+            Assert.AreEqual(person.CHI,ds.GetValue<string>(DicomTag.PatientID,0));
+            
+            // with anonymisation
+            generator.Anonymise = true;
+            
+            var ds2 = generator.GenerateTestDataset(person, r);
+
+            // we get a blank patient ID
+            Assert.IsTrue(ds.Contains(DicomTag.PatientID));
```

Comment:
ds not ds2? Already checked ds contains a PatientID tag in line 101, and haven't changed ds since that...